config/deploy.rb

```diff
@@ -38,10 +43,7 @@
 
   after :restart, :clear_cache do
     on roles(:web), in: :groups, limit: 3, wait: 10 do
-      # Here we can do anything such as:
-      # within release_path do
-      #   execute :rake, 'cache:clear'
-      # end
+      execute :bundle, :exec, "rake db:migrate RAILS_ENV=production"
```

Comment:
This seems to be a bit of overkill (do you have more than 3 app servers?).  Also, I *think* capistrano-rails [does this by default](https://github.com/capistrano/rails/blob/master/lib/capistrano/tasks/migrations.rake#L31).